print-battleground-state-changes

```diff
@@ -275,7 +275,7 @@ def json_to_summary(
     votes_remaining = expected_votes - votes
     precincts_reporting = row.precincts_reporting
     precincts_total = row.precincts_total
-    new_votes = 0 if last_iteration_info.votes is None else (votes - last_iteration_info.votes)
```

Comment:
`votes` here is not estimate votes, but all votes counted so far in the election, including third party and write-in candidates